corehq/pillows/mappings/xform_mapping.py

```diff
@@ -43,6 +43,10 @@
             "type": "date",
             "format": DATE_FORMATS_STRING
         },
+        "server_modified_on": {
```

Comment:
Do we actually put the mapping every time? For some reason I thought we only did that on index creation